### ISSUE TITLE
Animal starvation damage should be based on the game difficulty like player starvation

### ISSUE THREAD

**Issue by Adsper** [FossilsArcheologyRevival/FossilsArcheologyRevival#1258]
When the game difficulty is set to peaceful and an animal starves, hunger shouldn't matter, when set to easy, their health should drop to half, when set to normal it should drop to half a heart, and when on hard they should starve to death.

This could also probably be an additional hunger config option, with it just being static or changing based on difficulty, in case people change the game difficulty to test something and all their mistreated dinosaurs automatically starve to death in 2 seconds.

**Comment by Mike-U5** [comment]
I agree with a config option, but I disagree with dino's hungering faster on death. 
I always play on hard so mobs are actually a bit of a threat, but I wouldn't want dinosaurs to starve to death as a side-effect. That'd be incredibly obnoxious.
